src/OSPSuite.UI/Views/Importer/ColumnMappingView.cs

```diff
@@ -220,23 +220,13 @@ public override void InitializeBinding()
             columnMappingGridView.CloseEditor();
          });
 
-
-         _addButtonRepository.ButtonClick += (o, e) =>
-            OnEvent(() => _presenter.AddGroupBy(_gridViewBinder.FocusedElement.Source as AddGroupByFormatParameter));
          _disabledRemoveButtonRepository.Buttons[0].Enabled = false;
       }
 
       private RepositoryItem removeRepository(ColumnMappingDTO model)
       {
          if (model.Source is AddGroupByFormatParameter)
-         {
-            if (string.IsNullOrEmpty(model.Source.ColumnName))
-            {
-               return _disabledRemoveButtonRepository;
-            }
-
             return _addButtonRepository;
-         }
```

Comment:
@abdelr Not sure what happens here. Can u explain?

Reply:
The last column has the x icon for cleaning the row. For the special case of the add group by, it is meaningless so we added a plus instead (already configured in _addButtonRepository). So, just a visual change adding plus icon on add group by row, and keeping x icon for clear row

Reply:
ok, thx